core/src/test/java/ai/timefold/solver/core/impl/solver/termination/TerminationTest.java

```diff
@@ -57,7 +66,51 @@ void stepCountTerminationAtSolverLevel() {
         // 2 entities means 2 steps, but the step count limit is 1.
         // Therefore the best solution is uninitialized, but the score is still just zero.
         // (No init score.)
-        Assertions.assertThat(bestSolution.getScore()).isEqualTo(SimpleScore.ZERO);
+        assertThat(bestSolution.getScore()).isEqualTo(SimpleScore.ZERO);
+    }
+
+    @Test
+    void globalTimeSpentTerminationAtPhaseLevel() throws InterruptedException, ExecutionException {
```

Comment:
I suggest a second test case where the global termination is less than the LS time.